src/System.Runtime/tests/NSStatus/String.NSStatus.cs

```diff
@@ -0,0 +1,811 @@
+        string Local_56_20 = Param_56_00046df5.ToString();
+        string Local_56_21 = Param_56_00046df5.ToString();
+        string Local_56_22 = Param_56_00046df5.ToString();
+        string Local_56_23 = Param_56_00046df5.ToString();
+        string Local_56_24 = Param_56_00046df5.ToString();
+        string Local_56_25 = Param_56_00046df5.ToString();
+        string Local_56_26 = Param_56_00046df5.ToString();
+        string Local_56_27 = Param_56_00046df5.ToString();
+        string Local_56_28 = Param_56_00046df5.ToString();
+        object Param_56_000470db = (System.String)("ProgressBar");
+        string Local_56_30 = System.String.Format("{0}\u005c{1}",(object)Param_56_00046df5,Param_56_000470db);
+        string Expected_56 = "Themes\u005c{de3dbbcd-f642-433c-8353-8f1df4370aba}\u005cProgressBar";
+        temp = (Expected_56 == Local_56_30);
+        if(!temp) { Console.WriteLine(@"FAILED -> 56 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+
+        //*** System.String.Concat-0x72ae6765411ff31a-101 ***
+        temp = false;
+        // erid(0x829c1d6d83a38785
+        string Local_57_0 = System.String.Concat(new string[] {"https://","","","","app.vsspsext.visualstudio.com",":443","","/_apis/Profile/Profiles/me/Avatar","?size=Small",""});
+        string Expected_57 = "https://app.vsspsext.visualstudio.com:443/_apis/Profile/Profiles/me/Avatar?size=Small";
+        temp = (Expected_57 == Local_57_0);
+        if(!temp) { Console.WriteLine(@"FAILED -> 57 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+
+        //*** System.String.Compare-0x78f127d418523c57-101 ***
+        temp = false;
+        // erid(0x801318350923faf7
+        int Local_58_0 = System.String.Compare("AreChildItemsDisplayed","",(System.StringComparison)5);
+        int Expected_58 = 22;
+        temp = (Expected_58 == Local_58_0);
+        if(!temp) { Console.WriteLine(@"FAILED -> 58 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+
+        //*** System.String.Split-0x798e00162bc557f5-101 ***
+        temp = false;
+        // erid(0x8040b1acd0a6fe61
+        object Param_59_01efdf0b = (System.String)("https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/:::499b84ac-1321-427f-aa17-267ca6975798:::872cd9fa-d31f-45e0-9eab-6e460a02d1f1:::0");
+        string[] Local_59_1 = ((System.String)Param_59_01efdf0b).Split(new string[] {":::"},2147483647,(System.StringSplitOptions)0);
+        string[] Expected_59 = new string[] {"https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/","499b84ac-1321-427f-aa17-267ca6975798","872cd9fa-d31f-45e0-9eab-6e460a02d1f1","0"};
+        temp = (Expected_59.Length == Local_59_1.Length); for(int i=0;i<Expected_59.Length;i++){if(Expected_59[i] != Local_59_1[i]){temp = false; break;}}
+        if(!temp) { Console.WriteLine(@"FAILED -> 59 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+        Assert.Equal(errcount,0);
+    }
```

Comment:
> since their state should be independent.

Of course if it's not then these tests will be really hard to modify/maintain as any change or addition could pervert the behavior a subsequent test. Another reason for separate methods.